intel_models/icnet-camvid-fp32-0001/icnet-camvid-fp32-0001.md

```diff
@@ -0,0 +1,46 @@
+# icnet-camvid-fp32-0001
+
+## Use Case and High-Level Description
+
+This is the ICNet model that is designed to perform semantic segmentation. The model has been trained on the CamVid dataset from scratch using PyTorch framework. Training used median frequency balancing for class weighing. For details about the original floating point model, check out the [paper](https://arxiv.org/pdf/1704.08545.pdf).
+
+The model input is a blob that consists of a single image of "1x3x768x960" in RGB order. The pixel values are floating-point values that were obtained by taking each channel value as an integer in the [0, 255] range, dividing it by 255, and the normalizing the results given a mean of [0.39068785, 0.40521392, 0.41434407] and standard deviation of [0.29652068, 0.30514979, 0.30080369] (RGB order).
+
+The model output for `icnet-camvid-fp32-0001` is the per-pixel probabilities of each input pixel belonging to one of the 12 classes of the CamVid dataset.
+
+## Specification
+
+| Metric            | Value                 |
+|-------------------|-----------------------|
+| Type              | Semantic segmentation |
+| Source framework  | PyTorch               |
+
+## Accuracy
+
+The quality metrics calculated on CamVid validation dataset is 67.89% mIoU.
```

Comment:
Why do you specify metric value twice? The table with the result and specification on which dataset it was calculated, will be enough.

Also, on my view, specification how mean IOU was calculated also will not be superfluous (e.g.like [here](https://github.com/opencv/open_model_zoo/blob/develop/models/intel/semantic-segmentation-adas-0001/description/semantic-segmentation-adas-0001.md#accuracy))